policyengine_us/parameters/gov/states/mi/tax/income/credits/home_heating_credit/standard/allowance/additional_allowance_exemption.yaml

```diff
@@ -0,0 +1,11 @@
+description: Michigan home heating standard allowance increase this amount for each exemption over 6
+values:
+  2022-01-01: 182
+metadata:
+  period: year
+  unit: currency-USD
+  label: "Michigan home heating standard allowance increase per additional exemption"
+  reference:
+  - title: TABLE A - 2022 Home Heating Credit (MI-1040CR-7) Standard Allowance
+    href: https://www.michigan.gov/taxes/iit/accordion/credits/table-a-2022-home-heating-credit-mi-1040cr-7-standard-allowance
```

Comment:
we need a 2021 reference and the 2021 value

Reply:
all yaml files

Reply:
> we need a 2021 reference and the 2021 value
found 2021 instruction as value reference(table A), standard allowance yaml files await 2021 value
https://www.michigan.gov/-/media/Project/Websites/taxes/2022RM/IIT/BOOK_MI-1040CR-7.pdf?rev=d7990c15e0034d768a43c47b6d3ba0ac